bin/qless-js.js

```diff
@@ -210,11 +231,24 @@ commander
   .option('-d, --data <data>', 'Set the job data', JSON.parse)
   .option('-p, --priority <priority>', 'Set the job priority', parseInt)
   .option('-l, --delay <delay>', 'Delay the execution of the job by a number of seconds', parseInt)
-  .option('-t, --tags <tag>', 'Add a tag to the job', collect)
+  .option('-t, --tags <tag>', 'Add a tag to the job', collect, [])
   .option('-r, --retries <retries>', 'The number of retries to use for the job', parseInt)
-  .option('-s, --depends <depends>', 'Add a job on which it depends', collect)
+  .option('-s, --depends <depends>', 'Add a job on which it depends', collect, [])
   .action(put);
 
+commander
+  .command('recur <name> <next>')
+  .description('Put a job in the provided queue')
```

Comment:
Description seems to be inaccurate (it's the same as `put`)

Reply:
Copypasta